### ISSUE TITLE
Feature request: show information about current selection

### ISSUE THREAD

**Issue by Amunak** [pop4959/Chunky#177]
It would be nice if we could see the full current selection info - either with a command like `chunky selection`, or after any command that changes the selection. Ideally both.

Overall I'm not a big fan of having so many commands (I'd love to see subcommands like `chunky selection world|center|radius|shape` instead of a large list like we have currently, but that's a different issue.

**Comment by pop4959** [comment]
Sounds like a good idea. Right now there isn't really a way to check it without starting a task, which will then display stuff. I think ideally this also includes further information extrapolated from the selection, such as the block/chunk boundaries, total number of chunks, etc. How do you feel about that?

I'm not sure that I want to add another way to set the selection right now, as that would potentially add even more confusion, although I understand where you're coming from. I specifically considered a selection subcommand while initially designing the plugin but I also wanted to keep things from being too verbose.

Basically the goal was to replace this
```
worldborder world set 2000 2000 0 0
worldborder shape square
worldborder world fill 1000 0
worldborder fill confirm
```
with this
```
chunky radius 2000
chunky start
```

**Comment by Amunak** [comment]
> Right now there isn't really a way to check it without starting a task, which will then display stuff. I think ideally this also includes further information extrapolated from the selection, such as the block/chunk boundaries, total number of chunks, etc. How do you feel about that?

That'd be amazing actually.

> I'm not sure that I want to add another way to set the selection right now, as that would potentially add even more confusion, although I understand where you're coming from.

Yep, I agree.

WorldBorder doesn't have a very nice API either, especially because it isn't consistent. But I feel like you should've made a few "categories" of commands that belong together and otherwise keep it as is, which could be a nice middle ground. But changing it now wouldn't be ideal for existing users.

**Comment by jayberry14** [comment]
To add my suggestions to this issue, I think a command such as `chunky status`, or `chunky details` would be very important to a user so they don't necessarily need to start a new pre-generate job just to see what their current configuration is.

Info that I personally would like to see from this command might be the current world, the current radius, the current center, the current shape, how many blocks I have previously generated on this current world so I can more easily understand where to set my next boundary.

This list could go on, but I think it's sufficient for the most basic user as is

**Comment by pop4959** [comment]
Added a selection command to display the world, shape, center, and radius, in the latest commit.
